spec/features/admin/admin_modifies_items_spec.rb

```diff
@@ -47,23 +47,27 @@
     fill_in "Price", with: 67.95
     click_on "Update item"
 
-    expect(current_path).to eq(item_path(@item))
+    expect(current_path).to eq(admin_item_path(@item))
 
-    expect(page).to have_content "Item updated successfully!"
+    expect(page).to have_content "Item was successfully updated!"
     expect(page).to have_content "$67.95"
     expect(page).to_not have_content "$50.65"
   end
 
-  xscenario 'can modify retired status' do
+  scenario 'can modify retired status' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.